Build fails on Windows /w nightly

The output of `cargo build`:
```
> Executing task: cargo build <

   Compiling rusty-share v0.1.0
error[E0432]: unresolved import `OsStrExt`
  --> src\share_entry.rs:10:5
   |
10 | use OsStrExt;
   |     ^^^^^^^^ no `OsStrExt` external crate

warning: unused import: `OsStrExt`
  --> src\share_entry.rs:10:5
   |
10 | use OsStrExt;
   |     ^^^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

error[E0599]: no method named `as_bytes` found for type `std::ffi::OsString` in the current scope
  --> src\share_entry.rs:57:51
   |
57 |             percent_encoding::percent_encode(name.as_bytes(), percent_encoding::DEFAULT_ENCODE_SET)
   |                                                   ^^^^^^^^
   |
   = help: items from traits can only be used if the trait is in scope
   = note: the following traits are implemented but not in scope, perhaps
add a `use` for one of them:
           candidate #1: `use crate::os_str_ext::OsStrExt;`
           candidate #2: `use regex::input::Input;`
           candidate #3: `use clap::osstringext::OsStrExt3;`
```

And the output of `rustup show`:
```
Default host: x86_64-pc-windows-msvc

installed toolchains
--------------------

stable-x86_64-pc-windows-msvc
nightly-x86_64-pc-windows-msvc (default)

active toolchain
----------------

nightly-x86_64-pc-windows-msvc (default)
rustc 1.31.0-nightly (78ff609d7 2018-10-19)
```

I would like to fix this.

--- a/src/share_entry.rs
+++ b/src/share_entry.rs
@@ -2,12 +2,9 @@ use bytesize::ByteSize;
 use chrono::{DateTime, Local};
 use chrono_humanize::HumanTime;
 use crate::error::Error;
+use crate::os_str_ext::OsStrExt;
 use std::fs::DirEntry;
-#[cfg(not(target_os = "windows"))]
-use std::os::unix::ffi::OsStrExt;
 use url::percent_encoding;
-#[cfg(target_os = "windows")]
-use OsStrExt;
 
 #[derive(Debug)]
 pub struct ShareEntry {

